draft-irtf-t2trg-rest-iot.md

```diff
@@ -122,10 +124,12 @@ Since RESTful APIs are often simple and lightweight, they are a good fit for var
 The goal of this document is to give basic guidance for designing RESTful systems and APIs for IoT applications and give pointers for more information.
 
 Design of a good RESTful IoT system has naturally many commonalities with other Web systems.
-Compared to other systems, the key characteristics of many IoT systems include:
+Compared to other systems, the key characteristics of many RESTful IoT systems include:
 
 * need to accommodate for constrained devices, so with IoT, REST is not only used for scaling out (large number of clients on a web server), but also for scaling down (efficient server on constrained node)
 * data formats, interaction patterns, and other mechanisms that minimize, or preferably avoid, the need for human interaction
+* server endpoints are often highly constrained (e.g., simple sensor devices)
```

Comment:
at least some of the server endpoints...

Reply:
Yes. Should the text be tweaked? "some are often" would sound a bit weird :)

Reply:
"Often, at least some of the servers..." would also work.

Reply:
Maybe "often some classes of server endpoints (e.g., simple sensor devices) are highly constrained"?

Reply:
Latest incarnation:
> for some classes {{RFC7228}} of server endpoints, significant constraints, e.g., in energy consumption, and thus implementation complexity, may apply.